### DISCUSSION TITLE
Question: Official Ubuntu PPA?

### DISCUSSION THREAD

**Discussion by 0pLuS0** [DeaDBeeF-Player/deadbeef#2971] [General]
Hi,

I see on Launchpad you used it before, but there aren't any new updates on this.

https://launchpad.net/~alex-p/+archive/ubuntu/deadbeef

Did you stop using PPA, or is there still and Official PPA for Ubuntu?

THANKS

**Comment by Oleksiy-Yakovenko** [comment]
PPA packaging was provided by a volunteer / one of our slack/discord/irc members.
I was never tracking it closely. The guy who was doing probably lost interest.

**Comment by 0pLuS0** [comment]
Ahh ok...

THANKS
